python/cloudtik/runtime/spark/scripts/configure.sh

```diff
@@ -33,6 +34,7 @@ done
 function prepare_base_conf() {
     source_dir=$(cd $(dirname ${BASH_SOURCE[0]})/..;pwd)/conf
     output_dir=/tmp/spark/conf
+    SPARK_DEFAULTS=${output_dir}/spark/spark-defaults.conf
```

Comment:
I suggest not to put this line here. It will the meaning of prepare_base_conf since other runtime also have similar function. It would be better to keep implementation identical.

I suggest to move SPARK_DEFAULTS=${output_dir}/spark/spark-defaults.conf to configure_hadoop_and_spark right after the call of prepare_base_conf.

function configure_hadoop_and_spark() {
    prepare_base_conf
    SPARK_DEFAULTS=${output_dir}/spark/spark-defaults.conf